pnpm-lock.yaml

```diff
@@ -1,243 +1,308 @@
-lockfileVersion: 5.4
+lockfileVersion: '6.0'
```

Comment:
I don't think adding the lockfile changes is in scope of this PR

Reply:
Sorry but without this changes pipelines was failed

Reply:
Then we need to update/alter the pipeline or fix it separately.

Reply:
Should I cherry pick and provide another PR? In which branch i should merge it?

Reply:
@rschristian What should I do?

Reply:
@JoviDeCroock